### DISCUSSION TITLE
I Want to Shut Down bigrat.monster: Please leave your comment.

### DISCUSSION THREAD

**Discussion by SoNothingMC** [bigratmonster/bigrat.monster#72] [General]
> [!NOTE]
> Please read the [latest update](https://github.com/bigratmonster/bigrat.monster/discussions/72#discussioncomment-6865945) to the situation.

bigrat.monster's domain registration expires on ~~2023-07-11~~ _2025-07-11_ at 23:59:59 GMT. 

As I approach my 18th birthday, I think it's time to move on from the SoNothing alias and do something else. ~~So, I have decided to stop renewing the bigrat.monster domain.~~ _I have renewed the domain, but I don't think I want to do it again._

It's been a great experience over the past few years, goofing around different communities and social circles.  
If you have any comments or questions, please leave them here. This place will be the main discussion place for the future of the site.

Best Regards,  
SoNothing

**Comment by skyrina** [comment]
I am writing to you as a devoted follower and admirer of your remarkable creation, bigrat.monster. I cannot emphasize enough the profound impact this project has had on my life and the lives of countless others. It is with great admiration and a genuine desire for its legacy to continue that I humbly request your attention to this matter.

bigrat.monster has been an incredible source of inspiration and transformation for me personally. It has not only served as a captivating source of entertainment but also as a catalyst for personal growth and creativity. The unique blend of innovation, storytelling, and artistic expression embedded within the world of bigrat.monster has sparked my imagination and broadened my horizons.

Beyond its sheer entertainment value, bigrat.monster has fostered a sense of community and connection that is truly unparalleled. Through the Discord chats and events formed around this extraordinary creation, I have had the privilege of meeting like-minded individuals from diverse backgrounds who share a passion for this unique universe. The bonds forged through our mutual love for bigrat.monster have transcended geographical boundaries, age differences, and cultural barriers, creating a global community united by a common fascination.

As an avid supporter and active participant in the bigrat.monster community, I have witnessed firsthand the profound influence this project has had on fostering creativity, encouraging imagination, and building a strong sense of community.

Recognizing the immense impact that bigrat.monster has had on my life, I earnestly implore SoNothing to transfer over the domain, bigrat.monster, as it holds immense value and significance to me, and I believe that assuming responsibility for its ownership would enable me to contribute to the continued growth and preservation of this extraordinary creation.

I understand that transferring domain ownership is a significant decision, and I want to assure you that I am committed to upholding the legacy and integrity of bigrat.monster. I am prepared to fulfill any legal requirements, assume all associated costs and responsibilities, and provide the necessary resources to maintain and develop the domain effectively. I am also open to discussing any specific terms or conditions that SoNothing may have in relation to the transfer process.

I sincerely hope you will take my request into consideration and explore the possibilities of carrying on the legacy of bigrat.monster. The profound impact it has had on my life, and the lives of countless others, cannot be overstated. The rich tapestry of storytelling, the mesmerizing visuals, and the immersive experiences offered by bigrat.monster deserve to endure and inspire generations to come.

Thank you for your time and attention to this request. I look forward to hearing from you and discussing the possibility of transferring the domain ownership of bigrat.monster to me.

Yours sincerely,
Skylar

**Reply by SoNothingMC** [reply]
Thank you, Skylar. I am incredibly honoured to have my creation be recognised to have a profound impact on many individuals' lives.

By the sheer creativity that you have shown the community in the past few months, such as “yume twoki gameplay” and messages such as “meow meow mrow mrrp meew meow”, I am amazed at your ability to also enkindle the creativity of people around you. Especially those who, at times, may be in a bad state in their lives. You are able to bring people laughs and provide help to members in need. This is why you would be the perfect embodiment of what I want the successor of bigrat.monster to be.

That said, bigrat.monster is a project that is deeply associated with me. And unfortunately, as I enter the next stage of my life, I wish to simply wind down operations of the site, instead of passing the torch to another individual or group.

This is not an easy decision to make, and certainly not one that is made over-night. As the activity and operation of the surrounding community has already slowed down over the last year or so, I think this is the best decision to make. So, I am really sorry that I would have to deny your request to take over bigrat.monster. 

With appreciation,  
SoNothing

**Reply by skyrina** [reply]
Firstly, I want to express my sincere gratitude for your kind and heartfelt response.

While I completely understand and respect your decision to wind down operations of bigrat.monster as you enter the next stage of your life, I can't help but feel a sense of melancholy at the thought of its closure. The project has been a beacon of inspiration, bringing joy and connection to countless individuals. However, I trust that this decision was made after careful consideration and that you have the best interests of the community at heart.

I am grateful for the opportunity to have been a part of the bigrat.monster journey, even if only as a dedicated follower and supporter. The experiences, friendships, and memories forged within this community will forever hold a special place in my heart. I believe that while the project may come to an end, its impact and the legacy it leaves behind will continue to resonate with all those who have been touched by its magic.

Although I am saddened by the news of bigrat.monster's closure, I am genuinely grateful for the time and effort you invested in creating such a remarkable project. Your vision, creativity, and dedication have left an indelible mark on the lives of many. I would like to express my heartfelt appreciation for the joy, inspiration, and sense of belonging that bigrat.monster has provided.

As we bid farewell to this beloved project, I want to extend my best wishes to you as you embark on your new endeavors. May the future be filled with happiness, success, and fulfillment in all your endeavors. I will always cherish the memories and the impact that bigrat.monster has had on my life and the lives of others.

Thank you once again for everything you have done. I am grateful to have been a part of the bigrat.monster community and to have witnessed the magic it created.

With boundless adoration and immeasurable indebtedness that surpasses the boundaries of linguistic grandeur,
Skylar

**Reply by parknich** [reply]
didnt ask give me the domain

**Comment by lkd70** [comment]
no

**Comment by syrenaaa** [comment]
👋🐀

**Comment by SrIzan10** [comment]
bro i just discovered the domain and it's gonna shut down?
ok, you are going to represent yourself professionally but at least keep your domain and not talk about it.

**Reply by SoNothingMC** [reply]
Okay I admit I may have worded it wrong when I said I wanted to "be professional".  It's just that I want to drop this project as it's been taking up a whole lot of my life without actually doing much. It's always on the back of my mind, and it adds unnecessary stress.

**Reply by lkd70** [reply]
> Okay I admit I may have worded it wrong when I said I wanted to "be professional". It's just that I want to drop this project as it's been taking up a whole lot of my life without actually doing much. It's always on the back of my mind, and it adds unnecessary stress.

That's very understandable and relatable, do what's best for you. There's nothing to stop you coming back to this in the future when you need something to take your mind off of being a successful profrssional.

**Reply by Martenfur** [reply]
How exactly does hosting a png file take any time at all? Literally just set up hosting with Github Pages, point it to Cloudflare with your domain and you won't have to do anything, aside from paying, like, ten bucks every year or so. Gosh.

**Reply by SoNothingMC** [reply]
@Martenfur It’s people like you that makes me have to take up (stupid and unnecessary) energy to maintain the site and the surrounding community.

**Reply by parknich** [reply]
how? why do you need to maintain a site with a picture of a rat? just leave it be unless theres some random major security flaw (doubt it)

**Comment by ToxicAven** [comment]
Do professional people not do silly things?

**Reply by SoNothingMC** [reply]
It's not that I can't goof around anymore as a professional, but as part of (and after) the transition in the stages of my life (from the HS kid to a Uni student), I feel like I have better things to do than this.

**Reply by ToxicAven** [reply]
so transfer the site to someone else who is willing to keep it running

**Reply by skyrina** [reply]
I'm willing to keep it running

**Comment by Zly-u** [comment]
Tbh I don't really see how being professional and keeping such silly domain gonna impact your career or whatever.
I do YTPMVs and draw silly stuff, and I have a work as a programmer, nothing seemed to change for me.

**Reply by lkd70** [reply]
If anything it might help, can be used as an example of project management skills, etc...

**Reply by Zly-u** [reply]
Managing big rat is definitely hard, i'm not sure if I would manage that myself.

**Reply by SoNothingMC** [reply]
> I don't really see how being professional and keeping such silly domain gonna impact your career or whatever.

It's not about "being professional" that matters currently. Sorry for wording that wrong in the original post. What I ultimately want to do is to drop the SoNothing alias entirely.

> Managing big rat is definitely hard, i'm not sure if I would manage that myself.

Managing a domain is all fun and games until you have:
* Discord raid of people sending fucked up things
* People accusing you of credit card fraud
* Perverts in your chat
* …and more that I don't want to talk about anymore

So yeah, feel free to do that for 3 years straight, I am sure it is definitely very easy to do.

**Reply by Zly-u** [reply]
Well, handling communities and projects is a bit of different things, I would say.
And yeah i know how it is since i was a mod for 5+ years in the past lol, it's not fun, and not respected by people, sadly.

Either way, still unsure why would you close the domain, tho.

**Reply by boxofbytes** [reply]
just lock the discord if it's such a big problem

**Comment by bitfl0wer** [comment]
sorry we wont hire you because you host a big rat
STATEMENTS DREAMT UP ONLY BY FOOLS

**Reply by SoNothingMC** [reply]
Committing copyright infringement for 3 years straight isn't even the thing that will make me lose my career.  
My current "goof around" place (which I will also wind down my activity as I drop the SoNothing alias entirely) has said things so controversial, the product they make still gets downvoted to oblivion every time it gets mentioned on Reddit. Even after the claims were either debunked, or the person who said it has apologized. I believe that will jeopardize my future more than bigrat.monster.

**Reply by boxofbytes** [reply]
bigrat

**Comment by LappySheep** [comment]
bigrat

**Comment by My-Name-Is-Jeff** [comment]
WHERE 🐀

**Comment by BBaoVanC** [comment]
Are you transferring the domain to anyone or have any plan to continue hosting the images? My friends love this website because of its rat pictures that we can easily share with a cool link. If you don't want to host it yourself then I'm sure there's plenty of people out there that would happily host the media files instead.

My concern is that when it expires, someone else will squat it, or worse: use it for malware.

It is absolutely your decision to make, however I don't see the reason in either keeping the domain and leaving its website empty, or letting it expire without any way to let someone else rehost the big rat images at the domain we have become accustomed to.

**Comment by lenrik1589** [comment]
YOOOO that rat is amazing, wdym you want to shut it down, it's so cool! It shows that there are people in this world who still have it in them to do little things that could brighten up someone's day. It's just a wholesome picture of a rat, I don't think it is on any level "unprofessional" to just host such a cool and cute picture which i could send to someone "hey look, [here's a cool picture i found](https://bigrat.monster)" and it could brighten up their mood, but without it, that wouldn't be possible :<

**Comment by spark-slizer** [comment]
I tip my hat to you; one legend to another. - Rango (2011)

**Comment by thatbusinesscheems** [comment]
it's been a cheesy ride
![bigrat](https://github.com/bigratmonster/bigrat.monster/assets/75919679/998a1b89-7ae3-4984-b6cf-9fceb5d93c9a)

**Comment by tjb2640** [comment]
I will always remember Big Rat. This big guy was fundamental in transforming my views on modern technology.

**Comment by TheAtomicPunk** [comment]
what a shame, was a fun website. here's something that you may get a kick out of. recently, in every classroom ive been in at school i've written the link to bigrat.monster on the whiteboard repeatedly (https://cdn.discordapp.com/attachments/975770443388682292/1118223902796161155/image.png). i have gotten many teachers pissed off at me for this lmao. anyways i enjoyed your site while it was up, and will be sad to see it go.

**Reply by terrified9** [reply]
LOL

**Reply by segmentationfox** [reply]
In my school it's set as a wallpaper on all computers in like 3 computer rooms. And for some time in one of the classes changing a wallpaper was blocked in registry, so the big rats monsters were permanent.

![IMG_20230228_122611](https://github.com/bigratmonster/bigrat.monster/assets/70489677/c835677d-3ac7-47ed-ab90-723383085dd8)

**Reply by terrified9** [reply]
holy shit

**Comment by maestro-denery** [comment]
rest in peace. I'll miss this website every time I'll again send the link in various discord servers forgetting that it is shut down.

**Reply by terrified9** [reply]
same :pensive:

**Comment by majorsopa** [comment]
bruh.

**Comment by SoNothingMC** [comment]
Update 14/06: So I have made a (preliminary) compromise plan: I'll keep the domain alive for a couple more years to address @BBaoVanC's comment, but I'll freeze/archive the repository, so I no longer have to put any more energy into this. 

If I have more updates, I'll share them.  
Thanks for all of your support and comments.

**Reply by berkantdu123** [reply]
Bu-But I already had bought a domain to save the rat :( rest in peace https://bigrat.online

**Reply by Komos63** [reply]
Could you update the date for when the domain goes offline then?

**Reply by SoNothingMC** [reply]
> Could you update the date for when the domain goes offline then?

@Komos63 Hi, I have updated the date. Thanks for your reminder.

**Comment by Sunderw3k** [comment]
how much does the domain cost to renew?

**Comment by whupps** [comment]
rest in peace my silly rat monster fellow

**Comment by NOCKcorp** [comment]
its been a real honor

**Comment by ghost** [comment]
RIP bigrat.monster

**Comment by virtualtear** [comment]
Ima get the domain and keep up the legacy of the bigrat

**Comment by feeoodoor** [comment]
Bigrat will live on

**Comment by zhahngithub** [comment]
man this is so sad thanks bigrat i will never forget everything weve been through

**Comment by Doogie13** [comment]
Please at least do not let anyone malicious get the domain. Loved being a tiny part of this project :3

**Reply by terrified9** [reply]
fr

**Comment by qolarnix** [comment]
rewrite it in PHP

**Comment by adambrangenberg** [comment]
plz not thx

**Comment by void6670** [comment]
big rat will always be part of all us he will always have a place in my heart and everyones else, rest in peace big rat.monster you will be missed
![Screenshot_2023-06-21-13-05-19-682-edit_com android chrome](https://github.com/bigratmonster/bigrat.monster/assets/106617191/8154d18d-30de-4593-aa49-0c3bf1ab52d5)

**Comment by boxofbytes** [comment]
please don't do it please don't do kt pls j am beggging

**Comment by terrified9** [comment]
big rat is kil. no.

**Comment by terrified9** [comment]
please don't do i love big rat i can't live without big rat i htink i will hang myself without bigrat :sob: :sob: :sob: :sob: :sob: :sob:

**Comment by terrified9** [comment]
Dear bigrat.monster,

As I sit here, grappling with a wave of emotions, I find it difficult to put into words the depth of my sadness upon hearing the news of your imminent departure. It feels like bidding farewell to an old and cherished friend, one whose presence in my life has brought me immeasurable joy and inspiration.

From the moment I stumbled upon your virtual domain, I was captivated by the whimsical allure of your name. Bigrat.monster, you were a beacon of creativity and a sanctuary for all things extraordinary. Your pages were adorned with fantastical tales, vibrant illustrations, and a community that embraced the peculiar and the extraordinary with open arms.

You were more than a website; you were a realm of endless possibilities. Your virtual doors welcomed dreamers, artists, and seekers of magic, offering a respite from the mundane and a canvas for boundless imagination. Your stories sparked the flickering flame of inspiration within me, igniting a passion that will forever burn brightly in my heart.

Through your digital corridors, I discovered enchanting tales of daring quests, mythical creatures, and heroes who defied the odds. I marveled at the tapestries of words woven by talented writers, their prose painting vivid landscapes in my mind's eye. The illustrations that adorned your virtual galleries transported me to realms where reality intertwined with the extraordinary, where dreams danced upon the edge of the possible.

But now, as the digital curtain falls, I am left with a profound sense of loss. The impending silence where your virtual heartbeat once thrived echoes a melancholic tune, a requiem for a place that will forever hold a special place in my heart. Your departure leaves a void that cannot be easily filled, a void that serves as a reminder of the fleeting nature of the digital landscapes we hold dear.

While your pixels may fade, the memories we created within your virtual embrace will endure. The friendships forged, the tales shared, and the magic experienced will forever be etched in the collective memory of those who were fortunate enough to journey through your virtual corridors.

Farewell, dear bigrat.monster. May your pixels find new avenues to illuminate and inspire, and may the spirit of creativity and wonder you nurtured continue to thrive in the hearts and minds of all those who were touched by your digital embrace.

With a heavy heart and boundless gratitude,
Euro-pol

**Comment by PanTruskawka045** [comment]
https://bigrat.pantruskawka045.me (Suggest downloading that image)

**Comment by segmentationfox** [comment]
I created a mirror at https://bigrat.vercel.app if you want to use it. It doesn't have a cool domain name tho.. :(

**Comment by terrified9** [comment]
i've archived every page & subdomain of bigrat.

**Comment by SoNothingMC** [comment]
Locking the discussion for a bit… it seems like my updated comment has been auto-collapsed by GitHub (please *do* [read the comment](https://github.com/bigratmonster/bigrat.monster/discussions/72#discussioncomment-6176373).) and is not being shown to new commenters.

I'll update you all on any new progress. Thank you.

**Comment by SoNothingMC** [comment]
![bigrat.monster's domain registration has been extended to 2025.](https://github.com/bigratmonster/bigrat.monster/assets/32032824/ea7024ce-d606-4a46-9fb8-ac0afd3742fc)

and i oop

**Reply by ghost** [reply]
YAY

**Reply by terrified9** [reply]
YAY

**Reply by sogful** [reply]
AWESOME!!!!!!!

**Comment by liam-lemonade** [comment]
If this seemed to be a misreading, updating the README may help!

The loss of bigrat.monster was almost unfathomable, I'm glad so many people are will to carry the torch.

If not, leaving the server idle with public crowdfunding (domain & server costs) seems possible.

**Reply by SoNothingMC** [reply]
No, this is not a misreading — I manually renewed the domain for 2 years. My comment has been updated to reflect this. 
I will update the README (and the website description), but unfortunately I do not have access to my computer for a bit.

As for funding, I am not sure if public crowdfunding is even possible without revealing too much personal information… (ID card, home address, legal name, etc.) so I may have to continue paying for bigrat.monster out of pocket.

I would like to say that it is *absolutely fine* for me to pay for bigrat.monster. It is just that I am too emotionally tired to maintain this, but I don't want to give control to anyone else. (although letting the domain expire *is* letting other people control it… 😥)

**Reply by ToxicAven** [reply]
github sponsor?

**Reply by SoNothingMC** [reply]
> github sponsor?

@ToxicAven This option has the same issues of using a bank account, having income taxes & needing to disclose personal identities.

**Reply by sideloads** [reply]
Cryptocurrency, its widely available and people can buy it relatively easy using hot wallets.

**Comment by ghost** [comment]
YOU CANT TAKE HIM AWAY FROM ME
NOT ON MY BIRTHDAY
NOOOOOO

**Comment by terminite1** [comment]
Big ray is the best site I have ever come across. There is no other site that can compare with this site. It is simply majestic. I love big ray.

**Comment by BasToTheMax** [comment]
What is the renew price?

**Comment by ToasterXE** [comment]
bro please dont take it down, i discovered bigrat 2 years ago and have been showing it around all the time, giving around qr codes of bigrat or unpleasant gradient to my friends, it has become such an important part of my life and i really cant lose it right now. please reconsider your decision

**Comment by ToasterXE** [comment]
please bro

**Comment by Body-Alhoha** [comment]
if this shuts down i will send death threats to your family
for legal reason, this message is purely comical and is written by a 🐀

**Comment by 10ij** [comment]
Do you give permission to members of the community to rebirth this astronomical website?

**Reply by SoNothingMC** [reply]
No, unfortunately.

**Reply by terrified9** [reply]
Wtf r u gonna sue or some shit

**Reply by bytenekos** [reply]
There's a possibility of the domain getting used for nefarious reasons as said by bbaovanc [here](https://github.com/bigratmonster/bigrat.monster/discussions/72#discussioncomment-6148325)

**Comment by SoNothingMC** [comment]
# An Update

Apparently GitHub comments aren’t the best way to give people updated information, who would’ve thought?  
(this is referring to the fact that many people missed my previous since it was buried by other comments)

## Big Rat Lives Again… For Now

In case you haven’t seen it yet, I have extended the registration for bigrat.monster for the next two years. **The domain will now expire on 2025-07-11.**

Would I renew the domain again? Probably not, but that’s up to future me to decide.  
Everything could change in a span of two years, and that includes my potential decision.

## Finance

I would like to thank everyone who has asked about contributing to (or outright sponsoring!) bigrat.monster’s registration price.  
However, I have realised that it will take the same toll on me as managing a community, and therefore I have decided not to go forward with that plan.

<details><summary><i>Regarding Open Collective</i></summary>
The Open Collective collective <a href="https://opencollective.com/bigrat">bigrat.monster</a> is owned by me, but now it just lies dormant without a fiscal host.
</details>

## The Repositories

> [!IMPORTANT]
> This section includes important information regarding the big rat repositories. Stop scrolling and read it through.

~~Let’s set a date: October 31st November 19th, 2023.  
**On 12:00PM GMT of that day** (“contribution deadline”)**, all bigrat.monster-related repositories will be archived and no contributions will be accepted going forward.**~~

The Contribution Deadline has been extended indefinitely.

Before that, I will do another clean-up of the `/media` folder to remove unnecessary images.  
I plan for big rat to operate in a “bare-bones” mode after the archival, so nothing out-of-control happens.

## “SoNothing”

~~Although big rat will continue living for the next little while, the SoNothing alias will sunset as usual. I have already resigned from my Social Media Manager “internship” position over at [Cider](https://cider.sh), and I plan to slowly decrease my activity on this alias until it fades into nothing.~~

~~A detailed write-up of what’s going to happen will be published on <https://www.sonothing.com/bye> after I can allocate more time to write something (that is not a simple Markdown document like this). It’s going to show as a 404 page until I actually publish it.~~

This plan is also delayed indefinitely.

<details><summary><i>Actually, big rat is also going to fade into nothing.</i></summary>
<p>I currently manage the search results pointing towards big rat (where we surprisingly get a lot of traffic each month), when the time comes to shut down big rat (<b>NOT</b> the contribution deadline, this is far away in the future), you’ll see the repositories unarchive to remove <code>&lt;meta&gt;</code> tags so that it won’t be ranked as high.</p>

<p>3 months before the shutdown date, I will submit a request to search engines to remove big rat from search results.</p>

<p>On the date of the shutdown towards [unknown] years after the shutdown, bigrat.monster will have all its DNS records removed so nothing resolves.</p>

<p>When everything becomes a faded memory, the bigrat.monster domain will be released, and at that point, I will finally feel peace.</p>
</details>

## Merch (yep, you heard it right)

Before the Oct 30th contribution deadline, I might be able to take a few of my merch drafts online as a sort-of “parting gift”. I will try to keep the store up for as long as possible. (no maintenance required, a big plus)

As always, I will not receive any money from merch sales, and they will be automatically donated to [The Trevor Project](https://www.thetrevorproject.org/) without passing through my bank account.

**Update**: Since the contribution deadline has been extended indefinitely, new merch designs will still be on the table. Thank you for your support.

## big rat LLC Discord Server

Since the chat is dead anyways, I’ll just keep the server as-is.  
(Sorry, no new invites will be distributed.)

However, the ChargeHook bot will cease to function after the contribution deadline.

## Endword

I hope I addressed everyone’s concerns over this project. I am amazed at what people have created from this silly project of mine.

I hope this doesn’t become a cliché at this point, but I am really REALLY thankful for all of your support towards the big rat.

I hope to see you again soon – just in my IRL name.

**Reply by terrified9** [reply]
nice

**Reply by Nick-Nonya** [reply]
the big rat will live in our hearts and minds forevermore

**Comment by junaga** [comment]
keep it up <2

**Comment by pepeshki** [comment]
KEEP IT UP AHHHHH

**Comment by SoNothingMC** [comment]
## ℹ️ Contribution Deadline has been pushed back

hello and apologies, the contribution deadline will now be 19 November (still 12PM GMT).  
i am currently busy with university work and can't perform the necessary steps to properly clean up the repository.

again, i apologise about this. sowwwy 🥺

**Comment by KitenokGene** [comment]
R.I.P bigrat

**Comment by junaga** [comment]
How about you turn //bigrat.monster/ or another page, into a 404 page? I would redirect my 404 links into it unironically.

**Reply by BBaoVanC** [reply]
what is this even supposed to mean

**Reply by junaga** [reply]
Instead of showing a 404 page on my local website, i would redirect the request into a https://bigrat.monster/stylish-404-example-link.html

**Reply by BBaoVanC** [reply]
What would you want it to have on the page you redirect to?

**Reply by SoNothingMC** [reply]
This is such a bad idea lmao

**Reply by SoNothingMC** [reply]
Please do NOT redirect your error pages to bigrat.monster, if you really wanted the rat, just download the image and put it in your 404 page

**Comment by Nolram12345** [comment]
I fully believe that [bigrat.monster](https://bigrat.monster) is essential cultural heritage - at the least the main site and domain itself is. 
Perhaps it would be possible to trim content that doesn't serve the page's primary purpose (serving this image of big rat) but keep the page itself up? I think it provides great social & economic benefit to any human with internet access.

Kind regards,
a concerned visitor of [bigrat.monster](https://bigrat.monster)

**Reply by SoNothingMC** [reply]
Thank you for your kind words.

I do, in fact, want to do exactly what you said (see the *The Repositories* part of [this update comment](https://github.com/bigratmonster/bigrat.monster/discussions/72#discussioncomment-6865945)), though I was busy with school, so I just pushed the plans backwards indefinitely. I have a work-in-progress version of the [clean up](https://github.com/bigratmonster/bigrat.monster/tree/cleanup) on my local machine, and I'll push it up for a review when it's done.

Again, thank you for your support to bigrat.monster.
~ SN

**Reply by junaga** [reply]
dont delete plz

**Reply by SoNothingMC** [reply]
wont delete (at least in the near future) i promise 👍

**Reply by junaga** [reply]
https://twitter.com/possiblyazure/status/1792524230518526113?t=_Yh61v4dOKsp8pNeMlra0Q&s=19 hahahhaa

**Comment by junaga** [comment]
https://twitter.com/possiblyazure/status/1792524230518526113?t=_Yh61v4dOKsp8pNeMlra0Q&s=19

**Comment by AnotherPillow** [comment]
Did the registration get extended again? It seems to be expiring 11/7/2026 now.

**Reply by SoNothingMC** [reply]
Hi — It did! Registration prices went up late last year so I extended the domain before it.

**Comment by theotheroracle** [comment]
i love you big rat monster <3
